pdns/pdnsutil.cc

```diff
@@ -1197,6 +1197,24 @@ int createSlaveZone(const vector<string>& cmds) {
   return EXIT_SUCCESS;
 }
 
+int changeSlaveZoneMaster(const vector<string>& cmds) {
+  UeberBackend B;
+  DomainInfo di;
+  DNSName zone(cmds[1]);
+  if (!B.getDomainInfo(zone, di)) {
+    cerr<<"Domain '"<<zone<<"' doesn't exist"<<endl;
+    return EXIT_FAILURE;
+  }
+  vector<string> masters;
+  for (unsigned i=2; i < cmds.size(); i++) {
+    ComboAddress master(cmds[i], 53);
+    masters.push_back(master.toStringWithPort());
+  }
+  cerr<<"Updating slave zone '"<<zone<<"', master(s) to '"<<boost::join(masters, ",")<<"'"<<endl;
+  di.backend->setMaster(zone, boost::join(masters, ","));
```

Comment:
setMaster() can throw a PDNSException, it would be nice to wrap this call into `try{}/catch(){}`.